app/src/main/java/com/sdp/movemeet/Navigation/Navigation.java

```diff
@@ -41,6 +41,7 @@ public static void goToHome(View view) {
     public static void goToChat(View view) {
         Context mContext = view.getContext();
         Intent intent = new Intent(mContext, ChatActivity.class);
+        intent.putExtra("ACTIVITY_CHAT_ID", "general_chat");
```

Comment:
What is the purpose of this two strings as Extra ?

Reply:
The purpose is to send both the "Activity ID" and the "Activity Title" to the ChatActivity in order to properly create a dedicated chat for the sport activity. Once the user first lands on the newly created chat, he will be prompted with a welcome message in a TextView with the name of the sports activity corresponding to the chat.